test/infrastructure/docker/internal/docker/kind_manager.go

```diff
@@ -106,22 +103,22 @@ func (m *Manager) CreateWorkerNode(ctx context.Context, name, image, clusterName
 }
 
 // CreateExternalLoadBalancerNode will create a new container to act as the load balancer for external access.
-func (m *Manager) CreateExternalLoadBalancerNode(ctx context.Context, name, image, clusterName, listenAddress string, port int32, _ clusterv1.ClusterIPFamily) (*types.Node, error) {
+func (m *Manager) CreateExternalLoadBalancerNode(ctx context.Context, name, image, clusterName, listenAddress string, hostPort, cpPort int32, _ clusterv1.ClusterIPFamily) (*types.Node, error) {
 	// gets a random host port for control-plane load balancer
 	// gets a random host port for the API server
-	if port == 0 {
+	if hostPort == 0 {
 		p, err := getPort()
 		if err != nil {
 			return nil, errors.Wrap(err, "failed to get port for API server")
 		}
-		port = p
+		hostPort = p
 	}
 
 	// load balancer port mapping
 	portMappings := []v1alpha4.PortMapping{{
 		ListenAddress: listenAddress,
-		HostPort:      port,
-		ContainerPort: ControlPlanePort,
+		HostPort:      hostPort,
+		ContainerPort: cpPort,
```

Comment:
```suggestion
		ContainerPort: ControlPlanePort,
```
Let's use the existing const